src/test_utils.jl

```diff
@@ -498,19 +498,31 @@ function test_frule_correctness(
     # such that AD and central differences agree on the answer.
     x̄ = map(Base.Fix1(randn_tangent, rng), x_primal)
     ȳ = randn_tangent(rng, y_primal)
```

Comment:
Since `randn_tangent` is intended for testing, we could move the normalization logic into `randn_tangent`. This would also automatically fix the GPU test failures in #899.